routes/login/login.session.spec.js

```diff
@@ -88,8 +88,8 @@ describe('Test /login SESSION responses', () => {
     expect(firstError.text()).toMatch('Please enter an access code')
   })
 
-  // TEST RETURN TO SIN PAGE IF SIN IS WRONG
-  test('it returns a 422 on /login/sin page when the SIN does not match the access code', async () => {
+  // TEST REDIRECT TO ERROR PAGE IF SIN IS WRONG
+  test('it returns a 200 on /login/error/doesNotMatch page when the SIN does not match the access code', async () => {
```

Comment:
I think let's return the 422 if possible. Returning an error code is still the correct behaviour from an HTML point of view.

Reply:
We can still create the errors key and get a 422 status from `renderWithData` like we did on the `/sin` page: https://github.com/cds-snc/cra-claim-tax-benefits/pull/405/files#diff-dd564f1ad0228a52ea372878f1aec040R25

(The error object itself wouldn't be used for anything.)

Also, we can remove the 'errorsKey' from the sin get render method.

Reply:
Doing this doesn't appear to update the header location to the new URL - is that something we want to do? The URL stays on /dateOfBirth, and the tests are unable to check the header location.

Reply:
Well, actually that was something else I tried. Doing the above still puts a big scary red box at the top of the page asking for errors to be corrected.